packages/vm/test/t8n/ethereumjs-t8ntool.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [[ "$1" == "--version" ]]; then
+    echo "ethereumjs t8n v1"
+    exit 0
+fi
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+export NODE_OPTIONS="--max-old-space-size=4096" 
+tsx "$SCRIPT_DIR/launchT8N.ts" "$@"
```

Comment:
```suggestion
npx tsx "$SCRIPT_DIR/launchT8N.ts" "$@"
```

This way it will run even if I only have `tsx` installed locally within ethjs and not globally.